pre-docs/bolt_installing_modules.md

```diff
@@ -1,6 +1,40 @@
 # Installing modules
 
-Set up Bolt to download and install modules.
+Bolt is packaged with a collection of useful modules intended to support common work flows. The included modules are described in the [Puppetfile](https://github.com/puppetlabs/bolt/blob/master/Puppetfile) included in this repository. Modules and supporting documentation are publicly available on the [Puppet Forge](https://forge.puppet.com/).
```

Comment:
s/work flows/workflows/ - maybe? 
s/described/listed/ ?

Reply:
Will change to `workflows`.

I chose to use `described` to indicate that there is more than just a name, for example you can have a version, or information about a git repo/ref to download.